src/coreclr/vm/olevariant.cpp

```diff
@@ -134,48 +69,48 @@ VARTYPE OleVariant::GetVarTypeForCVType(CVTypes type)
 }
 
 //
-// GetCVTypeForVarType returns the COM+ variant type for a given
+// GetTypeHandleForVarType returns the TypeHandle for a given
 // VARTYPE.  This is called by the marshaller in the context of
 // a function call.
 //
 
-CVTypes OleVariant::GetCVTypeForVarType(VARTYPE vt)
+TypeHandle OleVariant::GetTypeHandleForVarType(VARTYPE vt)
 {
     CONTRACTL
     {
         THROWS;
-        GC_NOTRIGGER;
+        GC_TRIGGERS;
         MODE_ANY;
     }
     CONTRACTL_END;
 
-    static const BYTE map[] =
-    {
-        CV_EMPTY,           // VT_EMPTY
-        CV_NULL,            // VT_NULL
-        CV_I2,              // VT_I2
-        CV_I4,              // VT_I4
-        CV_R4,              // VT_R4
-        CV_R8,              // VT_R8
-        CV_DECIMAL,         // VT_CY
-        CV_DATETIME,        // VT_DATE
-        CV_STRING,          // VT_BSTR
-        CV_OBJECT,          // VT_DISPATCH
-        CV_I4,              // VT_ERROR
-        CV_BOOLEAN,         // VT_BOOL
+    static const int map[] =
```

Comment:
This is a minor nit, but something to consider. By converting this to `int`, we are making this static data 4 times bigger. Is that really needed?